src/components/Footer.js

```diff
@@ -13,7 +13,7 @@ const Footer = () => {
           <i className="fab fa-twitter"></i>
         </a>
       </div>
-      <p className="copyright">© 2022 AssetPoint</p>
+      <p className="copyright">© 2022 FinQuiz</p>
```

Comment:
- here instead of harcoding the current year we can use the date object to handle this for us.
```suggestion
      <p className="copyright">© {new Date().getFullYear()} FinQuiz</p>
```

Reply:
Thanks for good suggestions.
Implemented